design-system-docs/src/_component_examples/_foundations/typography_external_link.erb

```diff
@@ -0,0 +1,13 @@
+---
+title: External link
+iframe: false
+---
+
+<div class="cads-prose">
```

Comment:
Decided to be explicit about the `.cads-prose` element in the examples as well as the body copy, which is a change from the existing docs site.